js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -367,7 +367,7 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
       showBackfillSuccessToast(
         history,
         launchBackfillData?.launchPartitionBackfill.backfillId,
-        target.type === 'pureWithAnchorAsset',
+        true,
```

Comment:
Realized this should always be true now that we don't show any of these backfills in the runs page